app/src/main/java/org/sagebionetworks/research/mpower/tracking/fragment/MedicationDayFragment.kt

```diff
@@ -54,27 +54,27 @@ class MedicationDayFragment : AppCompatDialogFragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        var days : String? = null
+        var days: String? = null
         if (savedInstanceState == null) {
             if (arguments != null) {
-                schedId = arguments!!.getString(ARG_SCHED_ID)
-                name = arguments!!.getString(ARG_NAME)
-                time = arguments!!.getString(ARG_TIME)
+                arguments!!.getString(ARG_SCHED_ID)?.let { schedId = "" }
```

Comment:
Because arguments is within a null block, you can remove the "!!" operator on arguments and Android Studio should know it's a safe call.

Reply:
Fixed. The reason was `arguments` is actually a getter